app/controllers/v0/claim_documents_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module V0
+  class ClaimDocumentsController < ApplicationController
+    skip_before_action(:authenticate)
+
+    def create
+      attachment = klass.new(form_id: form_id)
+      # add the file after so that we have a form_id and guid for the uploader to use
+      attachment.file = params['file']
+      raise Common::Exceptions::ValidationErrors, attachment unless attachment.valid?
+      attachment.save
+      render json: attachment
+    end
+
+    private
+
+    def klass
+      case form_id
+      when '21-527EZ', '21P-530'
+        PersistentAttachment::PensionBurial
+      end
+    end
+
+    def form_id
+      (params[:form_id] || '21p-530').upcase # TODO: Remove default once frontend sends it.
```

Comment:
This will almost certainly be removed before merging - I think the frontend PR is getting in this morning. Just put it here initially because it was convenient to the problem I was solving that very second.